src/patternfly/components/MenuToggle/menu-toggle.scss

```diff
@@ -165,7 +165,7 @@
 
   position: relative;
   display: inline-flex;
-  align-items: center;
+  align-items: baseline;
```

Comment:
This could have some unexpected repercussions, too, based on what all is going inside of a menu toggle... curious this is needed?

Here is the big avatar menu toggle example - https://patternfly-pr-5700.surge.sh/components/menus/menu-toggle#with-avatar-and-text-full-height
<img width="734" alt="Screenshot 2023-07-10 at 5 22 10 PM" src="https://github.com/patternfly/patternfly/assets/35148959/b85c9fd5-c2e9-4c58-a6c9-3061054c8e1b">

It may just be that these menus are missing `.pf-m-full-height` (per the example title) but we'd still run the risk of a taller menu toggle (due to the content) having misaligned things in it.